index.js

```diff
@@ -169,12 +169,21 @@ function renderMedia (file, getElem, opts, cb) {
     tryRenderIframe()
   }
 
+  function addListener (elem, event, listener) {
+    elem.addEventListener(event, listener)
+    opts.onAddListener(elem, event, listener)
+  }
+  function removeListener (elem, event, listener) {
+    elem.removeEventListener(event, listener)
+    opts.onRemoveListener(elem, event, listener)
+  }
+
   function renderMediaSource () {
     var tagName = MEDIASOURCE_VIDEO_EXTS.indexOf(extname) >= 0 ? 'video' : 'audio'
 
     if (MediaSource) {
       if (VIDEOSTREAM_EXTS.indexOf(extname) >= 0) {
-        useVideostream()
+        return useVideostream()
```

Comment:
It's not ideal to return the `Videostream` instance in this case but to return nothing in the other cases. The user of this API will be very confused.